neuralmonkey/encoders/recurrent.py

```diff
@@ -103,6 +106,16 @@ def rnn_layer(rnn_input: tf.Tensor, lengths: tf.Tensor,
         if rnn_spec.cell_type == "LSTM":
             final_state = final_state.h
 
+    # residual commenctions
```

Comment:
wat? commenctions? :-D Tenhle komentář zahodit, ten if o řádek níž řiká úplně to samý